bundles/core-jdbc/src/main/java/org/orbisgis/corejdbc/ReadTable.java

```diff
@@ -96,22 +96,11 @@
                     if(!ascending) {
                         desc = " DESC";
                     }
-                    // Create a map of Row Id to Pk Value
```

Comment:
We do not return primary key in this function but row number. This is why I write this unit test. JTable RowModel is handling row id (0-n) not primary keys.